android/services/mongodb-remote/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/remote/internal/SyncMongoClientIntTests.kt

```diff
@@ -446,6 +446,11 @@ class SyncMongoClientIntTests : BaseStitchAndroidIntTest(), SyncIntTestRunner {
         testProxy.testVersionApplicationForEmptyVersionDocuments()
     }
 
+    @Test
+    override fun testFindOnReceivingChangeEvent() {
+        testProxy.testFindOnReceivingChangeEvent()
+    }
+
```

Comment:
[note] this changes position because I had it copied it from Tyler's repo and committed locally when I used it to fix the issue